src/renderer/entities/account/model/account.ts

```diff
@@ -22,13 +15,31 @@ export type Account = {
   cryptoType: CryptoType;
   chainType: ChainType;
   chainId?: ChainId;
-  // TODO: rename this to something as replacer for root account
-  isMain: boolean;
+  keyType?: KeyType;
+  type?: AccountType; // TODO: make required during wallet / account refactoring
```

Comment:
it won't be required because for Single Polkadot Vault, WalletConnect wallet it's always empty